editor/editor_inspector.cpp

```diff
@@ -1175,6 +1175,26 @@ void EditorInspectorSection::_notification(int p_what) {
 			draw_texture(arrow, Point2(Math::round(arrow_margin * EDSCALE), (h - arrow->get_height()) / 2).floor());
 		}
 	}
+
+	if (p_what == NOTIFICATION_DRAG_BEGIN) {
```

Comment:
Instead of storing the status with a notification, couldn't you use `gui_is_dragging()` as a Viewport function ? See https://docs.godotengine.org/fr/stable/classes/class_viewport.html#class-viewport-method-gui-is-dragging.

That would make the code simpler, as there is no other check done apparently.

Reply:
```c++
        if (p_what == NOTIFICATION_MOUSE_ENTER) {
		if (get_tree()->get_root()->gui_is_dragging()) {
			dropping_unfold_timer->start();
		}
	}

	if (p_what == NOTIFICATION_MOUSE_EXIT) {
		if (get_tree()->get_root()->gui_is_dragging()) {
			dropping_unfold_timer->stop();
		}
	}
```

Reply:
@groud @fire  thanks for looking at the PR and the review. I see a problem with this:

Let's say the user is dragging and dropping and the _release_ the mouse button on top of the section label. In this case, `MOUSE_EXIT` has not fired yet, but `DRAG_END` has. As a result, even if the drag is finished the unfold will still occur, even if they move their mouse outside of the InspectorSection area. This could be partially solved by just doing:

```gdscript
	if (p_what == NOTIFICATION_MOUSE_EXIT) {
		dropping_unfold_timer->stop();
	}
```

But the problem would still occur if they finish the drag and leave their mouse inside the section area. 

This is like... a really rare case, especially considering all this happens in 0.6 seconds, but it's still a slight inconsistency in behaviour. That's my thoughts - happy to change it if you guys think it's worth it.

Reply:
I don't think this bug blocks merging.

Reply:
Edited: Can you clarify is this a bug with this change or the existing pr?

Reply:
A bug with the suggested change.